docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -268,9 +268,157 @@ kube-dns-{network} shall be managed by Endpoints controller just like a regular
 
 __EndpointSlices__, the new type introduced in k8s v1.17, is out of current scope.
 
+### Network Policy
 
+By default, Arktos allows any pods to access others, across all tenants and namespaces. If network security is desired, network policy can be used to express the intents.
```

Comment:
Not sure if it's appropriate to state this way. In a production environment Arktos should work with a VPC network and it provides strong isolation.  Since flat network provider will be a CRD as well, not sure if it will be "default" :) And what kind of isolation will the flat network provide?

Reply:
doc updated to incorporate the feedback